src/utils/hide-balance/hide-balance.context.tsx

```diff
@@ -0,0 +1,15 @@
+import { createContext, ReactNode } from 'react';
+
+import { emptyFn, EmptyFn } from '../../config/general';
+
+interface HideBalanceContextValues {
+  isBalanceHidden: boolean;
+  balanceWrapper: (balance: ReactNode | string) => string;
```

Comment:
`EventFn<ReactNode | string>`

Reply:
here returns string, in EventFn by default returns void. maybe new type?

Reply:
you could extend `EventFn` with new generic type (with default value `void`)